fvtest/porttest/omrsockTest.cpp

```diff
@@ -1240,7 +1240,7 @@ TEST(PortSockTest, poll_functionality_many_sockets)
 	omrsock_socket_t clientSocket = NULL;
 	OMRSockAddrStorage connectedServerSockAddr;
 	omrsock_socket_t connectedServerSocket = NULL;
-	uint16_t port = 4930;
+	uint16_t port = 4931;
```

Comment:
Why is only this port number changed? There are 13 other uses of 4930 in this file; perhaps they should all be distinct?